.github/workflows/build-image.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and publish to Dockerhub
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  publish_to_dockerhub:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v2
+
+      - name: "Set up Docker Buildx"
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+
+      - name: "Log in to Docker Container Registry"
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
+
+      - id: get_version
+        name: "Get version from version.txt"
+        run: |
+          version=$(cat version.txt)
+          echo "::set-output name=version::$version"
+
+      - name: "Build and push Docker image"
+        uses: docker/build-push-action@v2
+        with:
+          context: ./stable/alpine
+          push: true
+          tags: cognite/docker-nginx-unprivileged:${{ steps.get_version.outputs.version }}
+          build-args: VERSION=${{ steps.get_version.outputs.version }}
```

Comment:
Do you still need this build argument for the image? As far as I can tell it is not used anywhere, but maybe I am missing something.

Reply:
fixed